bindings/python/symbols.cpp

```diff
@@ -101,6 +101,7 @@ void registerSymbols(py::module_& m) {
         .def_property_readonly("isScope", &Symbol::isScope)
         .def_property_readonly("isType", &Symbol::isType)
         .def_property_readonly("isValue", &Symbol::isValue)
+        .def_property_readonly("getDeclaredType", &Symbol::getDeclaredType)
```

Comment:
Since this is a property it should be named as a noun not a verb, i.e. `declaredType`